python/google/protobuf/json_format.py

```diff
@@ -379,11 +379,7 @@ def Parse(text, message, ignore_unknown_fields=False):
   """
   if not isinstance(text, six.text_type): text = text.decode('utf-8')
   try:
-    if sys.version_info < (2, 7):
-      # object_pair_hook is not supported before python2.7
-      js = json.loads(text)
-    else:
-      js = json.loads(text, object_pairs_hook=_DuplicateChecker)
+    js = json.loads(text, object_pairs_hook=_DuplicateChecker)
```

Comment:
Can you change the testDuplicateField in json_format_test.py as well? 
https://github.com/google/protobuf/blob/master/python/google/protobuf/internal/json_format_test.py#L793

Reply:
Sure. The PR is amended.